CHANGELOG.md

```diff
@@ -111,6 +111,8 @@ Main (unreleased)
 
 - Support Bearer tokens for metric remote writes in the Grafana Operator (@jcreixell, @marctc)
 
+- Fix issue when using a password file for `redis_exporter`. (@spartan0x117)
```

Comment:
Can you update the changelog entry [to describe what the issue was?](https://github.com/grafana/agent/blob/main/docs/developer/contributing.md#changelog-entry-style-tips)

Reply:
Done!